pynest/nest/lib/hl_api_nodes.py

```diff
@@ -92,19 +99,21 @@ def Create(model, n=1, params=None, positions=None):
                 layer_specs['center'] = positions.center
         if positions.extent is not None:
             layer_specs['extent'] = positions.extent
+        # For compatibility with SLI.
         if params is None:
             params = {}
         layer = sli_func('CreateLayerParams', layer_specs, params)
 
         return layer
 
-    params_contains_list = True
-    if isinstance(params, dict) and params:
-        params_contains_list = [is_iterable(v) or isinstance(v, Parameter)
-                                for k, v in params.items()]
-        params_contains_list = max(params_contains_list)
+    # If any of the elements in the parameter dictionary is either an array-like object,
+    # or a NEST parameter, we create the nodes first, then set the status. If not,
```

Comment:
Maybe `# or a NEST parameter, we create the nodes first, then set the given values. If not,`?